src/main/frontend/components/sidebar.cljs

```diff
@@ -461,13 +461,17 @@
   (let [finished (or (:finished state) 0)
         total (:total state)
         width (js/Math.round (* (.toFixed (/ finished total) 2) 100))
+        file-basename (util/node-path.basename
+                       (:current-parsing-file state))
+        display-filename (if (mobile-util/native-platform?)
+                           (js/decodeURI file-basename)
```

Comment:
`decodeURI` will raise `URI malformed` error with some chars, for example, `"https://test/ % s"`, there's a similar safe version of `decodeURIcomponent` [here](https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/util.cljs#L10).

Reply:
Replaced it with `js/decodeURIComponent`.